src/ucp/api/ucp.h

```diff
@@ -2017,6 +2029,70 @@ ucs_status_t ucp_rkey_ptr(ucp_rkey_h rkey, uint64_t raddr, void **addr_p);
  */
 void ucp_rkey_destroy(ucp_rkey_h rkey);
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Flags to define @ref ucp_request_desc_t::flags
+ *
+ * @ref ucp_request_desc_t::flags should be ORed by these flags to determine behavior
```

Comment:
@ref ucp_request_desc_t::flags is a combination of these flags  to determine behavior..